bpaingest/projects/stemcells/ingest.py

```diff
@@ -23,7 +23,7 @@
 import os
 import re
 
-logger = make_logger(__name__)
+logger = make_logger(__name__, logging.DEBUG)
 
```

Comment:
this doesn't seem to do anything? (see comment on your changes to `make_logger`)

Reply:
yeah think this just missed refactor (as I had other main set to INFO)